### PR TITLE
Modal Change Loomball during capture

### PR DESCRIPTION
**Pull request by andres123dbh**
- Show the user's loomballs in modal
- Color the border of the pressed loomball
- Update capture view and selected loomBall

### Note
- Depends on

### CODE REVIEW

**Comment by PedroChaparro** [comment]
Depends on [#97](https://github.com/PedroChaparro/loomies-backend/pull/97) now due to a field change

File: src/components/Modals/SelectLoomBall.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+export const SelectLoomBallModal = ({
+  isVisible,
+  toggleVisibilityCallback,
+  submitCallback
+}: iPropsSelectLoomBallModal) => {
+  const [loomBall, setLoomBall] = useState<TLoomball[]>([]);
+  const [selectLoomball, setSelectLoomball] = useState<TLoomball>();
+
+  //Get loomBall from user
+  const getUserLoomBalls = async () => {
```
Comment (PedroChaparro) [inline src/components/Modals/SelectLoomBall.tsx:24]: This is sending a request to the backend **every time the user changes the Loomball**. We don´t need that and and that can turn into a huge performance issue. 

I'll fix this, but be careful with the API calls and the `useEffect` hook.

File: src/components/Modals/SelectLoomBall.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  toggleVisibilityCallback: () => void;
+  submitCallback: (loomBall: TLoomball) => void;
+}
+
+export const SelectLoomBallModal = ({
+  isVisible,
+  toggleVisibilityCallback,
+  submitCallback
+}: iPropsSelectLoomBallModal) => {
+  const [loomBall, setLoomBall] = useState<TLoomball[]>([]);
```
Comment (PedroChaparro) [inline src/components/Modals/SelectLoomBall.tsx:20]: This names are confusing. Please, change 'em to loomballs, setLoomballs and selectedLoomball, setSelectedLoomball.

**Review by PedroChaparro — CHANGES_REQUESTED** [review]

**Review by PedroChaparro — APPROVED** [review]

**Review by Woynert — APPROVED** [review]
Good job!

**Pull request merged**